Five-Star-Technology/laravel-php-docker
# ELAB-597 Update for php 8.1

Updated docker to build for php 8.1

How to test
* cd into 8.1 dir
* run `docker build -t fivestartechnology/laravel-php-docker:8.1  .` and verify it builds correctly
